src/log-store/src/noop.rs

```diff
@@ -65,8 +67,8 @@ impl LogStore for NoopLogStore {
         Ok(AppendResponse { entry_id: 0 })
     }
 
-    async fn append_batch(&self, _ns: &Self::Namespace, _e: Vec<Self::Entry>) -> Result<Vec<Id>> {
-        Ok(vec![])
+    async fn append_batch(&self, _e: HashMap<Self::Namespace, Vec<Self::Entry>>) -> Result<()> {
```

Comment:
suggestion: We might need to allocate lots of small `Vec`. Since we could get the namespace id from the entry, could we pass `Vec<Self::Entry>`?
```suggestion
    async fn append_batch(&self, _entries: Vec<Self::Entry>) -> Result<()> {
```

Reply:
> suggestion: We might need to allocate lots of small `Vec`. Since we could get the namespace id from the entry, could we pass `Vec<Self::Entry>`?

Then we need to group entries by their namespaces inside `append_batch`.

Reply:
Does raft-engine have the restriction that entries in LogBatch are grouped by namespace?